.github/workflows/e2e-test.yml

```diff
@@ -33,8 +38,17 @@ jobs:
 
     - name: adding Vercel URL to Application's redirect uris
       env:
-        GITHUB_CONTEXT_EVENT: ${{ toJson(github.event) }}
         CONNECT_ACCOUNT_TEST_APP_ID: ${{ secrets.CONNECT_ACCOUNT_TEST_APP_ID }}
-        CONNECT_MANAGEMENT_URL: ${{ secrets.CONNECT_MANAGEMENT_URL }}
-        CONNECT_MANAGEMENT_API_KEY: ${{ secrets.CONNECT_MANAGEMENT_API_KEY }}
-      run: yarn e2e-add-redirect-uri
+      run: yarn e2e:add-redirect-uri
+
+    - name: creating test user
+      env:
+        CONNECT_TEST_ACCOUNT_EMAIL: ${{ secrets.CONNECT_TEST_ACCOUNT_EMAIL }}
+        CONNECT_TEST_ACCOUNT_PASSWORD: ${{ secrets.CONNECT_TEST_ACCOUNT_PASSWORD }}
+      run: yarn e2e:create-test-user
+
+    - name: removing test user
+      env:
+        CONNECT_TEST_ACCOUNT_EMAIL: ${{ secrets.CONNECT_TEST_ACCOUNT_EMAIL }}
+      run: yarn e2e:remove-test-user
+      
```

Comment:
No new line at the end of file.

Reply:
Don't understand the Github warning here. Last code line is 53, and there's an empty 54 line, I'm a bit confused.